modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -46,45 +58,47 @@ The repositories and channels for synchronizing upstream content must be configu
 In the {productname} {webui}, navigate to menu:Software[Channel List > All].
 Verify that you have a base channel and a child channel for your architecture.
 
-For example, for {ubuntu} 18.04:
-
-* Base channel: [systemitem]``ubuntu-18.04-pool for amd64``
-* Child channel: [systemitem]``Ubuntu-18.04-SUSE-Manager-Tools for amd64``
+For example, for {ubuntu} 20.04:
 
-.Procedure: Creating Custom Channels and Repositories
+* Base channel: [systemitem]``ubuntu-2004-amd64-main for amd64``
+* Child channel: [systemitem]``Ubuntu-20.04-SUSE-Manager-Tools for amd64``
 
+////
 You need to manually create three repositories:
 
 * For main
 * For main updates
 * For main security
 
-These examples are for {ubuntu} 18.04 (bionic).
+These examples are for {ubuntu} 20.04 (focal).
 Make sure you change the values to match the {ubuntu} version you want to use.
 
 In these procedures, we use ``http://archive.ubuntu.com/`` for {ubuntu} repositories.
 You can replace this URL with a more appropriate mirror.
 
+
+
```

Comment:
We don't need custom channels anymore for Ubuntu 20.04.

We need a mumble about this to clarify. I can't review and review the PR 20 times, so for now I will stop here.

Reply:
No, I started updating them, then realised and commented them out.

Reply:
as discussed, we don't need them for 20.04, but we need them for the other Ubuntu versions.